perun-registrar-lib/src/main/java/cz/metacentrum/perun/registrar/modules/EduGain.java

```diff
@@ -34,7 +35,12 @@ public Application approveApplication(PerunSession session, Application app) thr
 
 			AuthzResolver.setRole(session, user, vo, Role.TOPGROUPCREATOR);
 
-			Group membersGroup = session.getPerun().getGroupsManager().getGroupByName(session, vo, "members");
+			Group membersGroup = null;
+			try {
+				membersGroup = session.getPerun().getGroupsManager().getGroupByName(session, vo, "members");
+			} catch (InvalidGroupNameException e) {
+				// shouldn't happen, "members" is a valid name
```

Comment:
You should probably fail with ``InternalErrorException`` here, since module wants to add you as an admin of the members group, not just give you generic groupadmin role (with null group).

Reply:
Fixed.